src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -945,12 +945,12 @@ public virtual bool SupportsPaging(CallNode callNode, TexlBinding binding)
 
         // Returns true if function is row scoped and supports delegation.
         // Needs to be overriden by functions (For example, IsBlank) which are not server delegatable themselves but can become one when scoped inside a delegatable function.
-        public virtual bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata)
+        public virtual bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata, bool nodeInheritsRowScope = false)

```

Comment:
>= false [](http://example.com/codeflow?start=161&length=7)

Can we make this parameter required, to ensure that we pass the proper value everywhere?